javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1273,6 +1273,27 @@ public static boolean isAnonymousConstructor(final MethodTree method) {
         return false;
     }
 
+    /**
+     * Determines if the tree is a call to Arrays.copyOf()
+     *
+     * @param tree tree to check
+     * @return true if the given tree is a call to Arrays.copyOf() method
+     */
+    public static boolean isArrayscopyOfMethodInvocation(MethodInvocationTree tree) {
+        if (tree.getMethodSelect().getKind() != Kind.MEMBER_SELECT) {
+            return false;
+        }
+
+        MemberSelectTree memberSelectTree = (MemberSelectTree) tree.getMethodSelect();
+
+        if (memberSelectTree.getExpression().toString().equals("Arrays")
+                && getMethodName(memberSelectTree).equals("copyOf")) {
+            return true;
```

Comment:
This test isn't strong enough - it will return true on any method named `copyOf` in a class named `Arrays`, even if it is not the one from the JDK.

You should look at how `IndexMethodIdentifier` in the Index Checker identifies specific methods; you should be able to use the same kind of code here to identify `Arrays.copyOf` with confidence.

(It would probably also be worthwhile to add an inner static class named "Arrays" with a static "copyOf" method to the tests and then ensure that your code is not invoked when that method is called.)